.gitignore

```diff
@@ -46,3 +46,6 @@ gradle-app.setting
 
 # Rust build artifacts
 target/
+
+# File history folder
+/.history
```

Comment:
since this is just for the history plugin in your IDE, can you add this to your `~/.gitignore` instead?

Reply:
will do